servicecontrol/db-compaction.md

```diff
@@ -7,80 +7,39 @@ tags:
 ---
 
 ## Overview
```

Comment:
@indualagarsamy  I like the idea of retaining the secondary option although this get messy when we move to Raven 3 embedded and have to document a second UI.  But then we'll  also not be using teh system DB anymore so perhaps we'll need a whole separate doc for that.

Regarding  Clean Shutdown, yes the compact will fail if the DB isn't gracefully shutdown. That's my experience and  also what @boblangley  described.   There is a command line recovery option to get the DB into a clean state .  Based on this [thread](https://groups.google.com/forum/#!topic/ravendb/eKUYf2cHUkQ) it looks like we should also tell our users not to run repair prior to running recovery




.